Search/BingImageSearch/multiFeatured/.gitignore

```diff
@@ -0,0 +1,50 @@
+*.class
+
+# Auth filed
+*.auth
+*.azureauth
+
+# Mobile Tools for Java (J2ME)
+.mtj.tmp/
+
+# Package Files #
+*.jar
+*.war
+*.ear
+
+# Azure Tooling #
+node_modules
+packages
+
+# Eclipse #
+*.pydevproject
+.project
+.metadata
+bin/**
+tmp/**
+tmp/**/*
+*.tmp
+*.bak
+*.swp
+*~.nib
+local.properties
+.classpath
+.settings/
+.loadpath
+
+# Other Tooling #
+.classpath
+.project
+target/
+.idea
+*.iml
+
```

Comment:
it will be great if you can add the two file extensions that should not be included in this PR to be skipped as well by git